Services/CO.CDP.Person.WebApi/UseCase/ClaimPersonInviteUseCase.cs

```diff
@@ -37,7 +38,16 @@ public async Task<bool> Execute((Guid personId, ClaimPersonInvite claimPersonInv
         return true;
     }
 
-    private void GuardPersonInviteAleadyClaimed(PersonInvite personInvite)
+    private void GuardPersonInviteExpired(PersonInvite personInvite)
+    {
+        if (personInvite.ExpiresOn < DateTimeOffset.UtcNow)
```

Comment:
i think we'd need to use UtcDateTime of ExpiresOn in this comparison to make sure we're comparing UTC times here

Reply:
Yeah. Thats much better. I've applied this change.